samples/subsys/usb/uac2_explicit_feedback/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2023-2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+menu "UAC2 external feedback sample options"
+
+config USE_I2S_LRCLK_EDGES_COUNTER
```

Comment:
You can prefix it with APP_, like it was done for the MSC sample, if you want.

Reply:
Changed to `APP_USE_I2S_LRCLK_EDGES_COUNTER`.